### PR TITLE
Add more information about online and offline backup

### PR DESCRIPTION
**Pull request by AkshayGadhaveRH**
Currently the documentation does not talk about the usecases for the different types of backups. Users are bound to get confused about which procedure to use when backing up. Adding information about usecases for online and offline backups.

JIRA links:
- Backup documentation improvement: https://issues.redhat.com/browse/SAT-23077
- Online backups are for production: https://issues.redhat.com/browse/SAT-26312

#### Checklists

* [x] I am okay with my commits getting squashed when you merge this PR.
* [X] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [X] Foreman 3.12/Katello 4.14 (Satellite 6.16)
* [ ] Foreman 3.11/Katello 4.13
* [ ] Foreman 3.10/Katello 4.12
* [ ] Foreman 3.9/Katello 4.11 (Satellite 6.15; orcharhino 6.8/6.9/6.10)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* We do not accept PRs for Foreman older than 3.5.

### CODE REVIEW

**Comment by AkshayGadhaveRH** [comment]
@evgeni could you check and let me know if it looks good?

File: guides/common/modules/proc_performing-a-full-backup.adoc [hunk 1]
```diff
@@ -8,6 +8,9 @@ There are two main methods of backing up {ProjectServer}:
 * Offline backup
 * Online backup
 +
+With online backups, only services that affect the consistency of the backup are shut down while the backup process is running.
+For production usecases, use online backups.
```
Comment (evgeni) [inline guides/common/modules/proc_performing-a-full-backup.adoc:12]: So, when should I use offline ones? ;-)

**Review by asteflova — CHANGES_REQUESTED** [review]
Please @AkshayGadhaveRH add a link to the related Jira issue to the PR description.

You are adding information about both types of backup (online and offline) to the bullet point that describes one type (online). Please reconsider. The desired state is for each type of backup to be described under its corresponding label. In this case, those labels could be either the bullets on lines 8 and 9 or the discrete headings on lines 13 and 19.

File: guides/common/modules/proc_performing-a-full-backup.adoc (hunk 1 above)
Comment (AkshayGadhaveRH) [reply guides/common/modules/proc_performing-a-full-backup.adoc:12]: @evgeni should I remove this? I wasn't sure about this

Comment (evgeni) [reply guides/common/modules/proc_performing-a-full-backup.adoc:15]: I would, yeah.

Both backup methods are fine for production usecases, that was the point of the changes we implemented.

The tradeoff for the user is:
- When doing an offline backup, all services are down and we can just assume the data copied is consistent (= we perform no checks)
- When doing an online backup, only some services (which we know to produce inconsistencies) are down. As there still can be something interfering with the consistency, we perform checks (and may restart data-copy operations) which take additional time.

**Review by evgeni — APPROVED** [review]
Ack on the message, I'll leave the exact wording to Aneta!

File: guides/common/modules/proc_performing-a-full-backup.adoc [hunk 2]
```diff
@@ -6,8 +6,14 @@
 There are two main methods of backing up {ProjectServer}:
 
 * Offline backup
++
+All {Project} services are shut down during an offline backup.
+
 * Online backup
 +
+With online backups, only services that affect the consistency of the backup are shut down while the backup process is running.
```
Comment (ehelms) [inline guides/common/modules/proc_performing-a-full-backup.adoc:14]: This is rather vague from a user's perspective. Do we have a place where we go into more detail about what is possible and what is not possible during online backup capability wise?

Comment (evgeni) [reply guides/common/modules/proc_performing-a-full-backup.adoc:14]: We do not

Comment (AkshayGadhaveRH) [reply guides/common/modules/proc_performing-a-full-backup.adoc]: We can create a concept module for this with more detailed description.

Comment (ehelms) [reply guides/common/modules/proc_performing-a-full-backup.adoc]: Let's either do that as part of this or create a tracker to tackle it as a follow up.

**Comment by evgeni** [comment]
Small not on the cherry picks: please don't pick it further than 3.12, older releases do not have the online-backup improvements this description relies on.

**Review by asteflova — APPROVED** [review]

**Comment by evgeni** [comment]
To github.com:theforeman/foreman-documentation.git
   f818ad45f5..c9111f2173  3.12 -> 3.12

**Pull request merged**